boom.go

```diff
@@ -68,7 +80,8 @@ Options:
       metrics in comma-seperated values format.
 
   -m  HTTP method, one of GET, POST, PUT, DELETE, HEAD, OPTIONS.
-  -h  Custom HTTP headers, name1:value1;name2:value2.
+  -h  Semicolon-separated list of Custom HTTP headers, name1:value1;name2:value2.
```

Comment:
Maybe we should deprecate -h. It will break the existing users but they will get back on the docs to see the new -H flag.